linode/manager
# feat: [M3-7031] - Add AGLB Create Service Target Drawer

## Description 📝

- Adds the AGLB Create Service Target Drawer 🎉

## Preview 📷

![Screenshot 2023-09-14 at 10 54 52 PM](https://github.com/linode/manager/assets/115251059/98811aa3-ded9-4319-9b81-81c6c2268103)

## How to test 🧪
- Check out this PR
- Turn on the MSW
- Go to `http://localhost:3000/loadbalancers/0/service-targets`
- Click `Create Service Target`
- Test the drawer 🔍
  - The Linode Select and Certificate Select both are infinite scrollable and API filterable so your mileage will vary on the MSW
